static/beta/itless/modules/fed-modules.json

```diff
@@ -1,6 +1,7 @@
 {
     "landing": {
         "manifestLocation": "/apps/landing/fed-mods.json",
+        "fullProfile": false,
```

Comment:
This argument is deprecated, we use the `ssoScopes` config now with the full list of custom scopes.